server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,178 @@
+import textwrap
+import datetime
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from dash import callback, dcc, html
+from dash.dependencies import Input, Output
+
+from config import API_HOST
+from design import CONFIG_OPTIONS, DISCRETE_COLORS, DISCRETE_COLORS_MAP, LABELS, apply_figure_style
+
+title = "OVERVIEW COMBINED DASHBOARD"
+
+# DATA
+print(" * Downloading data for dataframe")
+query_string = '/reports?field=type_name&field=council_name&field=created_date'
```

Comment:
Let's give the `query_string`s distinct, meaningful names, instead of redefining it for every request.

Reply:
Also, you can use [`urlencode`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode) to build these URLs, instead of using a string literal. I think using `urlencode` will be more readable.

Reply:
Thanks for the suggestion - added the corresponding urlencode dictionaries.